docs/vtpm.md

```diff
@@ -0,0 +1,41 @@
+```ts
+ vTPM Object
+ {
+  id: string,
+  is_protected: boolean, // Currently unused by the XAPI, may be used in future.
+  is_unique: boolean // Currently unused by the XAPI, may be used in future.
+ }
+```
+
+- `is_protected`: The TPM is encrypted whenever it's transferred or at stored to disk (and exported)
+- `is_unique`: The TPM contents are exclusive to that VM, that means it has never been cloned (and maybe never exported nor imported)
+
+vTPM features are only enabled if XAPI is recent enough (check for `pool.restrict` or version (>= 8.3)).
+
+- vTPM
+  - At VM creation
+    - A toggle will be displayed in advanced settings under `boot firmware`.
+    - The default value is `false`, except for Windows >= W11.
+      - A message will be displayed below the toggle if:
+        - The value is `true`: _Please note that once the vTPM is used, its deletion is strongly discouraged as it will lead to data loss._
+        - The value is `false` and the VM is a Windows 11 or later: _VMs under Windows 11 or later cannot be created without vTPM_
```

Comment:
This must be based on the template defining the need for a vTPM or not, see above comment.